README.md

```diff
@@ -52,7 +52,7 @@ This project is built using the [Flask](http://flask.pocoo.org/) web framework.
 
 1. Copy the `.env.example` file to `.env`, and edit it including your credentials
    for the Twilio API (found at https://www.twilio.com/user/account/settings). You
-   will also need a [Twilio Number](https://www.twilio.com/user/account/phone-numbers/incoming) and TwiML App SID you made above.
+   will also need a [Twilio Number](https://www.twilio.com/user/account/phone-numbers/incoming) and TwiML App SID you made above. You will also need a DATABASE_URI (For example, using a Postgres database called `browser-calls-flask`, the DATABASE_URI would be `postgres:///browser-calls-flask`), and a SECRET_KEY. 
```

Comment:
I don't see where the SECRET_KEY is being used -- can it be removed?

Reply:
If someone chooses to use a postgres database, they'd also need to pip install psycopg2.

Reply:
The SECRET_KEY is used ... without it, you'll get an error in the terminal.

Reply:
How much detail do we want to get into with a chosen database? Do we want to include how to install, create a database, install psycopg2?

Reply:
In previous tutorials that we've done, I've seen a more opinionated stance on databases -- like the tutorial will just say "also install postgres" or "also install Mongo" with instructions for that particular db, and not go into other details.

Reply:
Also for the SECRET_KEY, I'm able to run the program without it -- the issue that I run into is if there's an empty env variable with `SECRET_KEY=` and no value, but if I remove SECRET_KEY entirely, things work.

No harm in keeping it in, but maybe just specify that the secret key can be anything/link to the Flask docs about what the secret key is? https://flask.palletsprojects.com/en/2.0.x/config/#SECRET_KEY

Reply:
Another alternative would be to add something like this:
```
--- a/browser_calls_flask/config.py
+++ b/browser_calls_flask/config.py
@@ -9,7 +9,7 @@ basedir = os.path.abspath(os.path.dirname(__file__))

 class DefaultConfig(object):
     SECRET_KEY = os.environ.get('SECRET_KEY', 'secret-key')
-    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URI')
+    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URI', 'sqlite:///' + os.path.join(basedir, 'prod.sqlite'))
     TWILIO_ACCOUNT_SID = os.environ.get('TWILIO_ACCOUNT_SID', None)
     TWILIO_API_KEY = os.environ.get('TWILIO_API_KEY', None)
```

And that way, we have a database option running, but if someone wants to use a different database, they can just set the DATABASE_URI environment variable. If there's no environment variable though, it'll use the sqllite db.